master/buildbot/www/rest.py

```diff
@@ -249,6 +249,15 @@ def checkFields(fields, negOk=False):
                 except Exception:
                     raise BadRequest('invalid offset')
                 continue
+            # As discussed with @tardyp, this is the best way to implement
```

Comment:
no need to put this comment here. But probably add some documentation about it in the rest part.